HPCC-15122 DOCS:Show Logical File Protection

g-pan opened the pull request:
FIX HPCC-15122 DOCS:Show Logical File Protection
Update the ECL Watch manual to show the new
Logical file protection icon.

Signed-off-by: G Panagiotatos greg.panagiotatos@lexisnexis.com
@JamesDeFabia please review
@miguelvazq please review

g-pan committed:
HPCC-15122 DOCS:Show Logical File Protection

FIX HPCC-15122 DOCS:Show Logical File Protection
Update the ECL Watch manual to show the new
Logical file protection icon.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
docs/images/ECLWA420.jpg this new image does not show the Logical file icon next to the logical filename.

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

g-pan committed:
HPCC-15122 DOCS:Show Logical File Protection

FIX HPCC-15122 DOCS:Show Logical File Protection
Update the ECL Watch manual to show the new
Logical file protection icon.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

g-pan committed:
HPCC-15122 DOCS:Show Logical File Protection

FIX HPCC-15122 DOCS:Show Logical File Protection
Update the ECL Watch manual to show the new
Logical file protection icon.

Signed-off-by: G Panagiotatos <greg.panagiotatos@lexisnexis.com>

JamesDeFabia commented:
Looks good to merge now

HPCCSmoketest commented:
Automated Smoketest Build: success
Install hpccsystems-platform-community_6.0.0-beta1.el6.x86_64.rpm
HPCC Start: OK
HPCC Stop: OK
HPCC Uninstall: OK

miguelvazq commented:
After speaking with you and reviewing, looks fine.
